avocado/core/task/statemachine.py

```diff
@@ -280,6 +280,14 @@ async def monitor(self):
                 runtime_task.status = 'FINISHED'
             except asyncio.TimeoutError:
                 runtime_task.status = 'FINISHED W/ TIMEOUT'
+                await asyncio.wait_for(self._spawner.terminate_task(runtime_task), None)
+                message = {'status': 'finished',
+                           'result': 'interrupted',
+                           'fail_reason': 'Test interrupted by SIGTERM: Timeout reached',
```

Comment:
Mentioning `SIGTERM` is misleading because it's not, at the very least, always being used.  I mean, a `process` based spawner may terminate the task with a `SIGTERM`, but that may not be the case for other spawners.

Reply:
Yes, makes sense, I will fix that.